spec/rspec/rails/matchers/active_job_spec.rb

```diff
@@ -252,4 +258,29 @@ def to_global_id(options = {})
       }
     end
   end
+
+  describe "have_been_enqueued" do
+    before { ActiveJob::Base.queue_adapter.enqueued_jobs.clear }
+
+    it "passess with default one number" do
```

Comment:
The default is ambiguous to me in this description. Is it `at_least` once or `exactly` once? This should be specific.